app/code/Magento/Quote/Api/Data/ShippingAssignmentInterface.php

```diff
@@ -34,6 +34,18 @@ public function getItems();
      */
     public function setItems($value);
 
+
```

Comment:
I would suggest refactoring this interface in case we are already introducing BiC, as it does not reflect the desired state described in the latest version of the Multishipping design